code/game/objects/items/storage/belt.dm

```diff
@@ -551,6 +551,15 @@
 	new /obj/item/ammo_magazine/pistol/vp78(src)
 	new /obj/item/ammo_magazine/pistol/vp78(src)
 	new_gun.on_enter_storage(src)
+	
+/obj/item/storage/belt/gun/m4a3/m1911/New()
+	..()
```

Comment:
`. = ..()`
Also why `m4a3/m1911` instead of just `m1911`? You shouldn't do it like that.

Reply:
Alright I'll change it to gun/m1911/New() and remember for future use.